apps/tangle-dapp/app/restake/unstake/types.ts

```diff
@@ -0,0 +1,6 @@
+export type UnstakeRequestTableData = {
+  uid: string;
+  amount: number;
```

Comment:
It's better to use serializable data for passing to the table. Since `BN` isn't serializable and `bigint` didn't work well, I switched to using `string`.